oneview/provider.go

```diff
@@ -53,7 +53,7 @@ func Provider() terraform.ResourceProvider {
 			"ov_apiversion": {
 				Type:        schema.TypeInt,
 				Optional:    true,
-				DefaultFunc: schema.EnvDefaultFunc("ONEVIEW_OV_API_VERSION", 200),
+				DefaultFunc: schema.EnvDefaultFunc("ONEVIEW_OV_API_VERSION", 0),
```

Comment:
what does this mean..?

Reply:
Default api version in the schema if the value is not found.  Keeping it 0, we are controlling what to for set the  api version